UnityXRContent/MARS/Templates/Platformer/Game_Advanced.unity

```diff
@@ -629,9 +629,9 @@ MeshCollider:
   m_Material: {fileID: 0}
   m_IsTrigger: 0
   m_Enabled: 1
-  serializedVersion: 3
+  serializedVersion: 4
```

Comment:
```suggestion
  serializedVersion: 3
```

Not sure if this is just the upgrade to 2019.4 but we should try to avoid changing serialized version